x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_define_rule/index.test.tsx

```diff
@@ -149,53 +150,6 @@ jest.mock('react-redux', () => {
 
 jest.mock('../../../../detections/containers/detection_engine/rules/use_rule_from_timeline');
 
-test('aggregatableFields', function () {
```

Comment:
I can see this function still exists.

Reply:
`aggregatableFields` is used internally in `DefineRuleStep` component. It's a trivial one line function where testing brings only maintenance burden. We should test only public interface and avoid exporting functions exclusively for testing.

Reply:
I don't think that in lack of test coverage for rule edit form, these tests are maintenance burden.
Has it been a case for this PR? I would just leave them in place